millicast/broadcast/software-encoders/obs/obs-enhanced-broadcasting-multitrack-video.mdx

```diff
@@ -19,7 +19,7 @@ import AbrImg from '../../../assets/img/842826436d56486b6a0a14b06bbcda1958658d3c
   <img src={AbrImg} width="600" />
 </div>
 
-To use this feature, _you will need OBS version `31.1` at a minimum_. Then, after choosing "Dolby Millicast" from the `Stream > Service`, enable _Enhanced Broadcasting_ e.
```

Comment:
![E](https://static.vecteezy.com/system/resources/previews/048/768/073/non_2x/cute-animal-alphabet-for-abc-book-e-letter-for-the-elephant-vector.jpg)